css/src/core/themes.scss

```diff
@@ -76,5 +76,11 @@ $theme-names: map.keys($themes);
 @each $key, $value in $themes {
 	.theme-#{$key} {
 		@extend %theme-base-styles;
+
+		@if $key == $root-theme {
+			color-scheme: light;
+		} @else {
```

Comment:
Would this work if we added a new theme? How might be do this in a way that could allow for flexibility in the future?

Reply:
Update approach